keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -215,13 +270,38 @@ new desired resources (i.e Spec.Containers[i].Resources.Requests) to the sum.
   further action is taken.
   - Kubelet retries the Pod resize at a later time.
 
+<<[UNRESOLVED]>>
+This is close to what I wrote above, but still vague.  What does it mean to
+"reject" a resize?  How do controllers know it has been deferred temporarily
+(will retry) or premantently (no more retries)? Does this put the pod into a
+new condition that higher-level controls need to know?
+<<[/UNRESOLVED]>>
+
 If multiple Pods need resizing, they are handled sequentially in the order in
 which Pod additions and updates arrive at Kubelet.
 
+<<[UNRESOLVED]>>
+I'd call this over-specced.  Kubelet should have freedom to choose higher-prio
+pods or "easy" cases or whatever.
```

Comment:
I agree - I was probably thinking FCFS just like admit.

However, eviction of lower-priority pods to facilitate resize was part of our early discussions, but we scoped it out to a future enhancement if really desired. It's not ruled out.

Reply:
sure, I'm happy to ignore this for now.  It's fine in the KEP, but I would make user-facing docs vague about policy.